django/db/backends/oracle/oracledb_any.py

```diff
@@ -1,20 +0,0 @@
-import warnings
-
-from django.utils.deprecation import RemovedInDjango60Warning
-
-try:
-    import oracledb
-
-    is_oracledb = True
-except ImportError as e:
-    try:
-        import cx_Oracle as oracledb  # NOQA
-
-        warnings.warn(
-            "cx_Oracle is deprecated. Use oracledb instead.",
-            RemovedInDjango60Warning,
-            stacklevel=2,
-        )
-        is_oracledb = False
-    except ImportError:
-        raise e from None
```

Comment:
This module is used in other places, e.g. `django.contrib.gis.db.backends.oracle`.